client/src/components/HabitForm/HabitForm.test.jsx

```diff
@@ -11,6 +11,7 @@ describe('HabitForm', () => {
 	const setup = fullMount => {
```

Comment:
Just a quick comment about test setup I've used, I see that you've renamed mount to `fullMount` which is a bit unnecessary IMO because, when set up like this, you'll need to import enzyme's mount above even if you're not using it.

Reply:
I realised that eslint was throwing an error `'mount' is already declared in the upper scope`. This means that if you do import `mount` from enzyme, the following `if (mount)` statement will resolve to true regardless if you actually wanted a full mount for that particular test or not.

Reply:
Damn, I missed that, tnx for noticing! I haven't seen that eslint error too for some reason :/

Reply:
Ah, I guess it gets passed `undefined` explicitly if there is no argument, but this makes it clearer. I made a commit that passes the `fullMount` function to the component on all client tests, instead of just `mount` so that you don't have to import mount if its not needed.